src/composables/useRouter.ts

```diff
@@ -0,0 +1,13 @@
+import { useRoute } from 'vue-router';
+
+import { computed } from 'vue';
+
+export function routeState() {
+  const route = useRoute();
+
+  const routeName = computed(() => route.name);
```

Comment:
What the need for this composable? Isn't `useRoute` good enough? We can use "route.name" instead of "routeName" isn't?

Reply:
Yeah, I thought it could save some lines over time and as we keep building it out more. Removed it for now